packages/components/src/schema/props/variant/input-password.ts

```diff
@@ -0,0 +1,29 @@
+import type { Generic } from 'adopted-style-sheets';
+
+import { watchValidator } from '../../utils/prop.validators';
+
+/* types */
+/**
+ * Loading-Passwordner
+ * - https://github.com/vineethtrv/css-loader
+ */
+const PasswordVariantPropTypeOptions = ['toggle', 'none'] as const;
+export type PasswordVariantPropType = (typeof PasswordVariantPropTypeOptions)[number];
+
+/**
+ * Defines the variant of Password navigation.
```

Comment:
Was ist mit "password navigation" gemeint?
Könntest du die prop noch mal konkreter beschreiben oder soll ich einen Vorschlag machen?

Reply:
ja, kannst du bitte einen Vorschlag machen?

Reply:
Gerne: `Defines the different variants for displaying the password input.`